### PR TITLE
Sundials/ML v6.1.1p1

### PR DESCRIPTION
**Pull request by tbrk**
Fix build under OCaml release candidates.

### CODE REVIEW

File: packages/sundialsml/sundialsml.6.1.1p1/opam [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		   "--os-distribution=%{os-distribution}%"]
+    [make "-j%{jobs}%"]
+    [make "doc"] {with-doc}
+]
+install: [
+  [make "install-findlib"]
+  [make "install-doc"] {with-doc}
+]
+depends: [
+    "ocaml" {>= "4.03.0" & < "4.14"}
```
Comment (mseri) [inline packages/sundialsml/sundialsml.6.1.1p1/opam:32]:
```suggestion
    "ocaml" {>= "4.03.0"}
```

i thiught this release is compatible with 4.14

**Comment by tbrk** [comment]
You're right @mseri. My bad! Thanks for fixing this.

**Comment by mseri** [comment]
It is still failing with
```
# lsolvers/sundials_nonlinearsolver_ml.c: In function 'custom_make':
# lsolvers/sundials_nonlinearsolver_ml.c:2080:13: error: "initialize" is deprecated: use "caml_initialize" instead [-Werror]
#  2080 |     ops->initialize      = (Field(vops, RECORD_NLSOLVER_OPS_INIT)
```

**Comment by mseri** [comment]
Looks good, thanks!

**Comment by tbrk** [comment]
Thanks to you!

**Pull request merged**